<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCounsellor_Dash">
    <property name="title">
     <string>Counsellor Dash</string>
    </property>
    <addaction name="actionBrand_Dash"/>
    <addaction name="actionModel_Dash"/>
    <addaction name="actionSales_Dash"/>
   </widget>
   <addaction name="menuCounsellor_Dash"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBrand_Dash">
   <property name="text">
    <string>Brand Dash</string>
   </property>
  </action>
  <action name="actionModel_Dash">
   <property name="text">
    <string>Model Dash</string>
   </property>
  </action>
  <action name="actionSales_Dash">
   <property name="text">
    <string>Sales Dash</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
